src/lib/VersionsCleaner.js

```diff
@@ -221,11 +237,18 @@ export default class VersionsCleaner {
     return { snapshot, version, diffString, first, diffArgs, record, skipVersion: shouldSkipVersion && reasonShouldSkipVersion };
   }
 
-  iterateSnapshots() {
+  iterateSnapshots(options = {}) {
+    const optionsAsArray = [];
+
+    if (options.from) {
+      optionsAsArray.push(`${options.from}..HEAD`);
+      optionsAsArray.push('--');
+    }
+
     return this.snapshotsRepository.iterate([
+      ...optionsAsArray,
       `${this.serviceId}/${this.documentType}.*`,
-      // For Multi page documents
-      `${this.serviceId}/${this.documentType} #*.*`,
+      `${this.serviceId}/${this.documentType} #*.*`, // For Multi page documents
```

Comment:
> As I understand this diff, this is just a minor correction to have the inline comment relate to the same line, with no functional change. This will make it harder to read the blame. I recommend to either correct the vocabulary, or avoid a churn altogether.
```suggestion
      `${this.serviceId}/${this.documentType} #*.*`, // for terms with multiple source documents
```